Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7.4-slim
+
+WORKDIR /srv/app
+
+COPY requirements.txt .
+
+RUN pip install --no-cache -r requirements.txt \
+    && rm -rf /tmp/*
+
+COPY . .
+
+RUN pip install -e .
+
+CMD "pytest -v"
```

Comment:
the [standard options](https://docs.docker.com/engine/reference/builder/#cmd) here are:
- `CMD pytest -v` (if you want to execute through a shell, e.g. using `$HOME` in an argument)
- `CMD ["pytest", "-v"] (if you don't need the shell)

Reply:
updated.